kolibri/plugins/user/assets/src/views/profile-page/index.vue

```diff
@@ -38,11 +38,11 @@
         {{ $tr('success') }}
       </ui-alert>
       <ui-alert
-        v-if="error"
+        v-if="unknownError"
         type="error"
         :dismissible="false"
       >
-        {{ errorMessage || $tr('genericError') }}
```

Comment:
Was this originally intended to handle an unknown error?

Reply:
Ah okay, can see that you moved the logic to the definition of `errorMessage`

Reply:
yup!